### PR TITLE
Add detailed release notes for 1.2

### PR DESCRIPTION
**Pull request by jakexks**
This is my initial effort at translating the release notes into a user-facing document

### CODE REVIEW

File: content/en/docs/release-notes/release-notes-1.2.md [hunk 1]
```diff
@@ -5,14 +5,84 @@ weight: 810
...
+  spec:
+    secretName: jks-keystore
+    jks:
+      create: true
+      passwordSecretRef:
+        name: supersecret
+        key: password
+  ```
+
+* ingress-shim now supports the new `cert-manager.io/usages` annotation for specifying custom key usages. If this isn't set, it now defaults to `server auth`, but a comma separated list of [any valid usages](https://pkg.go.dev/github.com/jetstack/cert-manager@v1.2.0/pkg/apis/certmanager/v1#KeyUsage) can be specified.
```
Comment (JoshVanL) [inline content/en/docs/release-notes/release-notes-1.2.md:56]: It maintains the same key usage default of Digital Signature and KeyEncipherment 

https://github.com/jetstack/cert-manager/blob/969b678f330c68a6429b7a71b271761c59651a85/pkg/apis/certmanager/v1/types.go#L209

**Review by JoshVanL — DISMISSED** [review]
Looks good. One thing from me.

File: content/en/docs/release-notes/release-notes-1.2.md (hunk 1 above)
Comment (jakexks) [reply content/en/docs/release-notes/release-notes-1.2.md:56]: 👍

Comment (jakexks) [reply content/en/docs/release-notes/release-notes-1.2.md:56]: Oh, it's never server-auth. I was mislead by the PR description

**Pull request merged**